R/analytics-app.R

```diff
@@ -5,13 +5,12 @@ analytics_ui <- function() {
     shiny::suppressDependencies("bootstrap"),
     shiny::suppressDependencies("plotlyjs"),
     style = "min-height: 100%;",
+    shiny::includeCSS(
+      system.file(
+        "examples", "app", "analytics", "www", "styles.css", package = "shiny.telemetry"
+      )
+    ),
```

Comment:
It has been a while since I worried about adding CSS to a shiny app because of Rhino. This works really well, and I think also very clear. Using `tags$link` I think requires that the `www` dir is at the root of the shiny app.